shared/src/business/entities/cases/Case.ts

```diff
@@ -1411,14 +1411,14 @@ export class Case extends JoiValidationEntity {
    * @param {object} caseDeadlines - the case deadlines
    * @returns {Case} the updated case entity
    */
-  updateAutomaticBlocked({ caseDeadlines }) {
+  updateAutomaticBlocked({ hasCaseDeadline }: { hasCaseDeadline: boolean }) {
```

Comment:
I changed `caseDeadlines` to `hasCaseDeadline` because 1) we don't actually need all of the case deadlines, 2) getting all of the case deadlines upon insert would have meant an extra db query following the upsert in `createCaseDeadline.ts`, and 3) passing in only a subset of deadlines (e.g., the case deadline upserted in `createCaseDeadline.ts`) seemed misleading to future devs.